### ISSUE TITLE
Add a visibility for the class constants

### ISSUE THREAD

**Issue by oliverklee** [mjaschen/collmex#137]
@mjaschen Should the constants be public? Or can they be protected (or private)?

**Comment by mjaschen** [comment]
I'd declare them public as long as we don't have explicit setters for attributes.

An example: to activate the delivery block flag for a customer, one can use 

```php
<?php
$customer->delivery_block  = Customer::NO_DELIVERY_BLOCK;
```

The same applies when using a non-magic setter:

```php
<?php
$customer->setAsInt('output_medium', Customer::OUTPUT_MEDIUM_EMAIL);
```

If we switch to specific setter methods, we can declare the constants private (or get rid of them by introducing a dedicated method per allowed value):

```php
<?php

$customer->setOutputMediumToPrint();
```

**Comment by oliverklee** [comment]
Okay. Thanks!

**Comment by oliverklee** [comment]
Closed with ce58a80accd8621cfd3c69347c2aec08984d492c.
